File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
import sys

while 1:
    answer='wrong'
    a,b,c  = map(int,sys.stdin.readline().split())
    if a==b==c==0:
        break
    if a*a + b*b == c*c:
        answer='right'
    if a*a + c*c == b*b:
        answer='right'
    if b*b + c*c == a*a:
        answer = 'right'
    print(answer)

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.util.*;

public class Main {
    public static void main(String[] args) {  
        try (Scanner sc = new Scanner(System.in)) {
           while(true){
               int a = sc.nextInt();
               int b = sc.nextInt();
               int c = sc.nextInt();
               String answer = "wrong";
               if(a==0 && b==0 && c==0) break;
               if(a*a+b*b==c*c) answer="right";
               if(b*b+c*c==a*a) answer="right";
               if(a*a+c*c==b*b) answer="right";
               System.out.println(answer);
           }
        }
    }
}
